src/ImageSharp/ApplyProcessors.cs

```diff
@@ -0,0 +1,107 @@
+﻿// <copyright file="RotateFlip.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp
+{
+    using System;
+
+    using ImageSharp.PixelFormats;
+
+    using ImageSharp.Processing;
+
+    /// <summary>
+    /// Extension methods for the <see cref="Image{TPixel}"/> type.
+    /// </summary>
+    public static partial class ImageExtensions
+    {
+        /// <summary>
+        /// Mutates the image by applying the operations to it.
+        /// </summary>
+        /// <typeparam name="TPixel">The pixel format.</typeparam>
+        /// <param name="source">The image to rotate, flip, or both.</param>
+        /// <param name="operations">The operations to perform on the source.</param>
+        public static void Mutate<TPixel>(this Image<TPixel> source, Action<IImageOperations<TPixel>> operations)
```

Comment:
We should be consistent:
- Either by redesigning our whole API to be fully extension method focused, so not only `Mutate()`, but all the methods/properties producing derived results from a narrow core API, should be extension methods. So even `image.Bounds` would become an `image.Bounds()` extension method.
- Or just turn both `Mutate(operations)` and `Clone(operations)` to instance methods as well